components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -958,6 +958,11 @@ KeyringController::SignMessageByDefaultKeyring(
 }
 
 void KeyringController::AddAccountsWithDefaultName(size_t number) {
+  if (!default_keyring_) {
+    DCHECK(false) << "Should only be called when default keyring exists";
```

Comment:
1) `DCHECK(false)` --> `NOTREACHED()`

2) as per spec, this is an anti-pattern, see https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++.md#check_dcheck_and-notreached
though personally I'm okay to keep it as is

Reply:
This should be
DCHECK(default_keyring_) << "Should only be called when default keyring exists";

Reply:
We still need the surrounding `if` condition because for release channel we want to return early there. So I think it's the same.  I think with the message is slightly better than `NOTREACHED`